docs/index.rst

```diff
@@ -17,6 +17,7 @@
    cli
    api
    citations
+   governance/ROADMAP
```

Comment:
It seems that adding the ROADMAP to the docs (a good idea! :+1:) should be its own PR that is separate from the maintenance going on here.

Reply:
Can't. It causes a warning->error.

Reply:
Fair.